second-gen/packages/swc/.storybook/preview.ts

```diff
@@ -29,6 +29,7 @@ const preview = {
             },
         },
     },
+    tags: ['autodocs'],
```

Comment:
Automatically triggers documentation pages for every component. Components can opt-out using `tags: ['!autodocs']`